migration/migration.go

```diff
@@ -349,6 +349,9 @@ func BootstrapWorkItemLinking(ctx context.Context, linkCatRepo *link.GormWorkIte
 	if err := createOrUpdateWorkItemLinkType(ctx, linkCatRepo, linkTypeRepo, link.SystemWorkItemLinkPlannerItemRelated, "One planner item or a subtype of it relates to another one.", link.TopologyNetwork, "relates to", "is related to", workitem.SystemPlannerItem, workitem.SystemPlannerItem, link.SystemWorkItemLinkCategorySystem); err != nil {
 		return errs.WithStack(err)
 	}
+	if err := createOrUpdateWorkItemLinkType(ctx, linkCatRepo, linkTypeRepo, link.SystemWorkItemLinkTypeParentChild, "One planner item or a subtype of it parent of another one.", link.TopologyNetwork, "parent of", "child of", workitem.SystemPlannerItem, workitem.SystemPlannerItem, link.SystemWorkItemLinkCategorySystem); err != nil {
```

Comment:
We probably want to also use the link topology "tree" (see [here](https://www.visualstudio.com/en-us/docs/work/reference/link-type-element-reference#link-type-topologies-and-restrictions]). It is currently not allowed by the design folder but we certainly need it sooner than later.